model-optimizer/extensions/ops/DetectionOutput.py

```diff
@@ -36,18 +37,21 @@ def __init__(self, graph: Graph, attrs: dict):
             'input_height': 1,
             'normalized': 1,
             'share_location': 1,
+            'clip_after_nms': 0,
+            'clip_before_nms': 0,
+            'decrease_label_id': 0,
```

Comment:
May be make default values to be `False` instead of `0`?

Reply:
Yes, updated.